### ISSUE TITLE
Toolbar yellow color is too high (not an issue)

### ISSUE THREAD

**Issue by pasqualedante** [tmacharia/pine-gap#3]
Hi @tmacharia this isn't an issue but is related only to personal taste. From my point of view, the yellow in toolbar (solution explorer and other windows) is too high related to other (dark) colors. Maybe should be closer to yellow used for class. If you not agree, please close this issue as invalid. I will continue to use your theme anyway.
Thanks.

**Comment by tmacharia** [comment]
Hi dante, I will give it a closer second look.

**Comment by mayureshs** [comment]
I agree with @pasqualedante. The yellow can use a little lighter/dimmed shade. I am talking about the yellow background in the solution explorer/toolbar.
